backstop-non-responsive-ce.json

```diff
@@ -391,6 +391,53 @@
       "misMatchThreshold": 0,
       "requireSameDimensions": true
     },
+    {
+      "label": "input-text",
+      "cookiePath": "",
+      "url": "http://localhost:8000/tests/custom-elements/text-input.html",
+      "referenceUrl": "",
+      "readyEvent": "backstop_ready",
+      "readySelector": "",
+      "delay": 0,
+      "hideSelectors": [],
+      "removeSelectors": [],
+      "hoverSelector": "",
+      "clickSelector": "",
+      "postInteractionWait": "",
+      "selectors": [
+        ".test-default",
+        ".test-default-icon-left",
+        ".test-default-icon-right",
+        ".test-default-icon-left-right",
```

Comment:
@janibolkvadze missing ".test-default-label-inline" and ".test-default-label-with-icon" . Please add them to the tests.

Reply:
Good catch, done.